lib/matplotlib/cm.py

```diff
@@ -328,7 +328,8 @@ def to_rgba(self, x, alpha=None, bytes=False, norm=True):
         treated as an RGB or RGBA array, and no mapping will be done.
         The array can be `~numpy.uint8`, or it can be floats with
         values in the 0-1 range; otherwise a ValueError will be raised.
-        If it is a masked array, any masked elements will be set to 0 alpha.
+        Any nans will be set to 0 alpha and if *x* it is a masked array, any
+        masked elements will be set to 0 alpha.
```

Comment:
Might want to just simplify the sentence and remove the "if it is a masked array" qualifier.
```suggestion
        Any nans or masked elements will be set to 0 alpha.
```